bb/logmsg.c

```diff
@@ -14,7 +14,7 @@
 
 static loglvl level = LOGMSG_WARN;
 static int do_syslog = 0;
-static int do_time = 1;
+static int do_time = 0;
```

Comment:
No timestamp by default? What about adding "logmsg notimestamp" to legacy default?

Reply:
You find those helpful?  The default right now depends on whether the database runs with a tty.  Can certainly keep the default and disable with legacy defaults.

Reply:
I kinda like the timestamps personally, but maybe I only want them for debugging purposes. So I guess actually it doesnt really matter